starter-code/css/main.css

```
@@ -6,3 +6,136 @@ Green: #00B172
 White: #FFF
 
 */
+
+#main-logo{
+    position: absolute; top: 15px; left: 40px;
+    width: 20%;
+}
+
+#menu {
+ background-color: white;
+ width: 100%;
+ height: 60px; 
+}
+
+#menu li {
+    list-style-type:none;
+    font-size: 22px;
+    font-family:  Arial;  
+    width: 80px;
+    height: 60px;
+    float: right;
+    margin: 25px 50px;  
+}
+
+#menu li:hover{
+    color:#00B172;
+}
+
+.bloque-1{
+    background-image: url("../images/landing.jpg");
+    background-size: cover; 
+    width: 100%;
+    height: 650px;  
+}
+
+.inner-block {
+    position: relative; top: 100px;
+}
+
+h1{
+    color: white;
+    width: 100%;
+    z-index: 10;
+    text-align: center;
+    font-size: 50px;
+    font-family: Arial;
+}
+
+.bloque-1 p{
+    color: white;
+    z-index: 10;
+    text-align: center;
+    font-size: 30px;
+    font-family: Arial;
+}
+
+h3 {
+    color: black;
+    text-align: center;
+    font-size: 30px;
+    font-family: Arial;
+    border-bottom-style: solid;
+    border-bottom-color: #00B172; 
+    border-bottom-width: 10px 1px;
+}
+
+#contenido {
+ position: absolute; top:870px;   
+ background-colo: white;
+ width: 100%;
+ height: 400px; 
+}
+
+#contenido li{
+    list-style-type:none;
+    font-size: 22px;
+    font-family:  Arial;  
+    width: 200px;
+    height: 60px;
+    float: left;
+    margin: 25px 100px;
+   }
+
+h4 {
+    color: #00B172;
+}
+   
+.iconos {
+    width: 100%;
+}
+
+.bloque-2 {
+    position: absolute; top:1350px;
+    background-color: #00B172;  
+    width: 1260px;
```

En otros dispositivos que no sean tu lap, esto se ve extraño, te sugiero que le coloques `100vw` (100 viewport width) para que tome el 100% del ancho de la pantalla.
```suggestion
    width: 100vw;
```